content/src/controller/Controller.ts

```diff
@@ -304,34 +313,65 @@ export class Controller {
     const entityTypes: (EntityType | undefined)[] | undefined = stringEntityTypes
       ? stringEntityTypes.map((type) => this.parseEntityType(type))
       : undefined
+    //deprecated
     const fromLocalTimestamp: Timestamp | undefined = this.asInt(req.query.fromLocalTimestamp)
+    // deprecated
     const toLocalTimestamp: Timestamp | undefined = this.asInt(req.query.toLocalTimestamp)
+    const from: Timestamp | undefined = this.asInt(req.query.from)
+    const to: Timestamp | undefined = this.asInt(req.query.to)
     const offset: number | undefined = this.asInt(req.query.offset)
     const limit: number | undefined = this.asInt(req.query.limit)
+    const lastId: string | undefined = req.query.lastId?.toLowerCase()
 
     // Validate type is valid
     if (entityTypes && entityTypes.some((type) => !type)) {
       res.status(400).send({ error: `Found an unrecognized entity type` })
       return
     }
 
+    // TODO: remove this when to/from localTimestamp parameter is deprecated to use to/from
+    const fromFilter = from ?? fromLocalTimestamp
+    const toFilter = to ?? toLocalTimestamp
+
     const requestFilters = {
       entityTypes: entityTypes as EntityType[] | undefined,
-      fromLocalTimestamp,
-      toLocalTimestamp
+      from: fromFilter,
+      to: toFilter
     }
     const { pointerChanges: deltas, filters, pagination } = await this.service.getPointerChanges(
       requestFilters,
       offset,
-      limit
+      limit,
+      lastId
     )
     const controllerPointerChanges: ControllerPointerChanges[] = deltas.map((delta) => ({
       ...delta,
       changes: Array.from(delta.changes.entries()).map(([pointer, { before, after }]) => ({ pointer, before, after }))
     }))
+
+    if (controllerPointerChanges.length > 0 && pagination.moreData) {
+      const lastPointerChange = controllerPointerChanges[controllerPointerChanges.length - 1]
+      pagination.next = this.calculateNextRelativePathForPointer(lastPointerChange, pagination.limit, filters)
+    }
+
     res.send({ deltas: controllerPointerChanges, filters, pagination })
   }
 
+  calculateNextRelativePathForPointer(
+    lastPointerChange: ControllerPointerChanges,
+    limit: number,
+    filters?: PointerChangesFilters
+  ): string | undefined {
+    const nextFilters = Object.assign({}, filters)
```

Comment:
Is this copy necessary? You are not modifying `nextFilters`

Reply:
True!